app/views/holdings/index.html.erb

```diff
@@ -1,31 +1,36 @@
 <p id="notice"><%= notice %></p>
 
-<h1>Holdings</h1>
+<h1>Portfolio</h1>
 
+<h3>Total Portfolio Worth</h3>
+
+<h3>Stocks</h3>
 <table>
   <thead>
     <tr>
-      <th>User</th>
-      <th>Company</th>
-      <th>Quantity</th>
-      <th colspan="3"></th>
+      <th>Name</th>
+      <th>Symbol</th>
+      <th>Shares</th>
+      <th>Current Price</th>
+      <th>Average Cost</th>
+      <th>Total Return</th>
+      <th>Equity</th>
     </tr>
   </thead>
 
   <tbody>
     <% @holdings.each do |holding| %>
       <tr>
-        <td><%= holding.user_id %></td>
-        <td><%= holding.company_id %></td>
+        <td><%= holding.company_name %></td>
+        <td><%= holding.company_ticker %></td>
```

Comment:
company_name && holding.company_ticker are invalid. Do you mean 
`holding.company.name` 
&& 
`holding.company.ticker`? 

To properly join, you might also need to add 
`
belongs_to :user
belongs_to :company
`
to models/holding.rb